exporter/prometheus/prometheus.go

```diff
@@ -123,11 +96,8 @@ func (o *Options) onError(err error) {
 // corresponding Prometheus Metric: SumData will be converted
 // to Untyped Metric, CountData will be a Counter Metric,
 // DistributionData will be a Histogram Metric.
+// Deprecated in lieu of metricexport.Reader interface.
 func (e *Exporter) ExportView(vd *view.Data) {
```

Comment:
Not sure if it's the right way to go to simply remove the support for view data. From what I saw Stackdriver exporter still supports both: https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/master/stats.go#L173.

Reply:
For existing user of SD, they have to explicitly add a call to StartMetricExporter and StopMetricExporter. So to avoid breaking, support for ExportView was left as is. In case of prometheus, there is no need to for ExportView anymore because Gatherer can always read using new interface. There is no additional code change required to enable collecting metrics using reader interface.